vaadin-text-field-flow-integration-tests/src/test/java/com/vaadin/flow/component/textfield/tests/BinderValidationPageIT.java

```diff
@@ -76,21 +76,21 @@ public void fields_internalValidationPass_binderValidationFail_fieldInvalid() {
 
     @Test
     public void fields_internalValidationPass_binderValidationFail_validateClient_fieldInvalid() {
-        fieldClasses.forEach(clazz -> {
```

Comment:
Why did you change that test to be applied only for TextArea?

Reply:
Temp changes. Test is bad because doesn't say which component failed.
Had to detect it and check only `TextArea`.

Reverted, thanks.